### ISSUE TITLE
Generator persists in making sounds until the block is removed or the client relogs

### ISSUE THREAD

**Issue by utoc** [McJty/DeepResonance#113]
Good day

We noticed an issue in DR this morning when a user finally had the resources to get into DR.  When the generator is started, the space-theme sound effect starts but after the generator is switched off, the sound persists until the client relogs or the generator block is broken.  This appears to be happening in SSP and SMP.

We are running FML 2073
 DR 1.10-1.2.1beta38
 rftools 1.10-5.17 and
 mcjtycore 1.10-1.9.9

Please let me know what other information I can provide to assist

**Comment by McJty** [comment]
Hmm strange. I will investigate

**Comment by McJty** [comment]
I cannot reproduce this. How exactly was the generator turned off? What were the exact steps to reproduce this?

**Comment by utoc** [comment]
Image: http://i.imgur.com/3hq0g8B.png

Generator has a lever on the front or sides, doesn't matter.  I shorted the multiblock down to just a generator and a generator controller and had the same results.

SSP Image: http://i.imgur.com/5tS3boh.png

After some extensive debugging; this ends up being a crossmod interaction with.. Botania?  Removed it and sure enough, the noise persisting forever behavior goes away.

I'm not sure where to take the debugging process from here; its a little baffling.
